next.config.js

```
@@ -1,5 +1,154 @@
+    optimizeFonts: {
+      inlineFonts: true,
+      preload: true,
+      subsets: ['latin'],
+    },
+    
+    // Enable incremental cache for better performance
+    incrementalCacheHandlerPath: require.resolve('./cache-handler.js'),
+    
+    // Enable server components external packages
+    serverComponentsExternalPackages: ['@radix-ui/react-dialog'],
+  },
```

Biome (1.9.4)

[error] This property value named experimental is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named experimental


(lint/suspicious/noDuplicateObjectKeys)